PyInstaller/building/makespec.py

```diff
@@ -369,6 +373,8 @@ def main(scripts, name=None, onefile=None,
 
     hiddenimports = hiddenimports or []
 
+    exclude_upx = exclude_upx or []
+
```

Comment:
Better remove the blank line above. This keeps the for terse, and both lines are similar.